project/Yu-Rou-Weng/proposal.rst

```diff
@@ -0,0 +1,169 @@
+.. enough for people to remember.
+
+.. Additional documents or files may be used.  They should be added in the
+.. repository.
+
+Basic Information
+=================
+
+.. Create a GitHub repository to host your project and add the URL here.  The
+.. GitHub repository has an 'About' field, in which you are encouraged to write a
+.. simple statement (preferably one sentence) to introduce the project.
+
+The conversion of NASA's Plot3D to SU2 file format pertains to mesh data conversion.
+
+In the realm of mesh data conversion, numerical solutions for partial differential
+equations (PDEs) rely on the discretization of space. The entities that describe this
+discretized space are known as grids or meshes.  Here are 3 types of codes standing out 
+to conducting effective numerical analysis: (i) a mesh generator creates the grid, (ii) 
+a solver computes the solution, and (iii) a visualizer provides the analysis. Among these 
+components, mesh data holds serve as a central element connecting these codes.
+
+Converting NASA's Plot3D to SU2 file format is a key aspect of this mesh data conversion 
+process, allowing seamless integration and utilization of the numerical solutions within 
+different computational frameworks.
+
+Problem to Solve
+================
+
+.. Describe the problem or the set of problems you want to solve.  Include
+.. necessary background information without making it lengthy.
+
+.. Some points may help you organize the problem description:
+
+.. 1. The field or industry of the problem.
+.. 2. The physics and/or the mathematics behind the problem.
+.. 3. The algorithm or numerical method that should be applied for solving the
+   problem.
+
+Plot3D and SU2 server as good examples of the numerical analytical codes:
+
+1.Plot3D is a file format which can only store structured grid and result data and 
+  remains a only file format in the filed of Computaional Fluid Dynamic(CFD).
+
+2.SU2 is a suite of open-source software tools written in C++ for the numerical solution 
+  of partial differential equations (PDE) and performing PDE-constrained optimization.
+
+When dealing with numerical analysis, the mesh data format differs between Plot3D and SU2. 
+Plot3D uses structured mesh as input, while SU2 uses unstructured mesh for solving fluid dynamics.
+
+The goal of this project is to create a file format converter that can translate Plot3D mesh data 
+into a format compatible with SU2. This converter will function as a standalone executable, facilitating 
+data exchange between different programs(codes). Also, the converter will be integrated to [modmesh](https://github.com/solvcon/modmesh)
+to serve as a useful plugged-in library for easier automation.
+
+Prospective Users
+=================
+
+.. Describe the users of your software and how they will use it.  It is OK to
+.. combine this section with the previous one (`Problem to solve`_).
+
+1.Modmesh: (Will add more explanation after discussions)
```

Comment:
It will be good to add a sentence or two for how do you speculate the integration with modmesh.